src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenidConnectAuthenticationHandler.cs

```diff
@@ -235,180 +276,215 @@ protected override AuthenticationTicket AuthenticateCore()
             {
                 IFormCollection form = await Request.ReadFormAsync();
                 Request.Body.Seek(0, SeekOrigin.Begin);
-
-                openIdConnectMessage = new OpenIdConnectMessage(form);
+                message = new OpenIdConnectMessage(form);
             }
 
-            if (openIdConnectMessage == null)
+            if (message == null)
```

Comment:
Sad decision :sob:

GET support was probably the most wanted feature concerning the OIDC client middleware (at least on JabbR). You must keep in mind that `form_post` is not supported everywhere (e.g Google or Ping Identity with some `response_type`).